frontend/src/global_styles/layout/_breadcrumb.sass

```diff
@@ -30,7 +30,7 @@
   @include default-font(var(--breadcrumb-font-color), var(--breadcrumb-font-size), bold)
 
 #breadcrumb,
-.wp-breadcrumb
+.op-wp-breadcrumb
```

Comment:
This is a big file with a lot of stuff going on, so it might be hard. But to have clean BEM, I reccommend you just add a new file (for example, `_wp-breadcrumb.sass`) and then do all the `op-wp-breadcrumb` stuff there nicely and cleanly.

If you changed all the breadcrumbs in the application, then I would remove this file alltogether.